app/account/views.py

```diff
@@ -250,3 +250,9 @@ def unconfirmed():
     if current_user.is_anonymous() or current_user.confirmed:
         return redirect(url_for('main.index'))
     return render_template('account/unconfirmed.html')
+
+
+@account.route('/donate')
+def donate():
```

Comment:
you should probably add an `@login_required` to this route. Check out some of the other routes in this file for examples.